docfx/doxygen2markdown.cc

```diff
@@ -633,6 +643,89 @@ bool AppendIfListItem(std::ostream& os, MarkdownContext const& ctx,
   return true;
 }
 
+// The `variablelist` element type is defined as a sequence of "groups".
+// Groups do not create an XML element, they are simply a description of
+// "element A is followed by element B". This requires some funky processing.
+//
+//   <xsd:complexType name="docVariableListType">
+//     <xsd:sequence>
+//       <xsd:group ref="docVariableListGroup" maxOccurs="unbounded" />
+//     </xsd:sequence>
+//   </xsd:complexType>
+//
+//   <xsd:group name="docVariableListGroup">
+//     <xsd:sequence>
+//       <xsd:element name="varlistentry" type="docVarListEntryType" />
+//       <xsd:element name="listitem" type="docListItemType" />
```

Comment:
So one of these things can't have multiple `listitem`s?

If they could, then the `nested.paragraph_indent = ...` below could introduce unwanted `- `s

Reply:
I may be missing the intent of your question.  These are supposed to have exactly one `varlistentry` and exactly one `listitem`, yes.  I think the code is fine.

Reply:
That answers my question.